build_bot/common/cmd_utils.py

```diff
@@ -0,0 +1,63 @@
+import os
+import subprocess
+import tempfile
+
+
+def shell(cmd, env=None):
+    print("calling command: " + str(cmd))
```

Comment:
this project not associated with the Web Part. it's a separate module. We need to think about how best to do it. The modules should be independent